e2e-tests/Cargo.toml

```diff
@@ -5,11 +5,16 @@ members = [
     "gas_benchmark",
     "host_functions_test",
     "keylet_exists",
+    "test-utils",
```

Comment:
should probably use `snake_case` to match the rest of this workspace

Reply:
> Also, does this need to be a member of the workspace?

Yeah, probably just for `cargo` to work.